rules/embeds/vimeo.js

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2022 DigitalOcean
```

Comment:
```suggestion
Copyright 2023 DigitalOcean
```
(Can you make sure its also 2023 in any other files this PR touched?)